data_portal/models.py

```diff
@@ -1068,6 +1079,20 @@ def get_by_keyword(self, **kwargs) -> QuerySet:
 
         return qs
 
+    def get_library_by_workflow_keyword(self, **kwargs):
+        qs: QuerySet = self.all()
+
+        type_name = kwargs.get('type_name', None)
+        if type_name:
+            qs = qs.filter(workflows__type_name__iexact=type_name)
+
+        status_list_string = kwargs.get('end_status', None)
```

Comment:
This is supporting a multi-value string for the `end_status`?
What's the use case?

Reply:
Thinking of using it as the status filter bar as the client could select multiple statuses

![image](https://user-images.githubusercontent.com/61998484/137404472-96b49f45-d31e-442c-9130-1f48c5578d08.png)

Reply:
Perhaps for very specific requests we could consider a POST with a query payload?

Reply:
Yep; as discussed, let stick to single value `status=foo` for now. We will revisit when use case stronger for multi-values.